core/src/banking_stage/transaction_scheduler/scheduler_metrics.rs

```diff
@@ -0,0 +1,400 @@
+use {
```

Comment:
more copypasta? :pray: :pleading_face:

Reply:
Original PR changed some metrics, which caused conflict since file didn't exist. I did a bad merge resolve - this file should not exist and is not used. 🪓

Reply:
OH HEY THAT WAS HALF OF THE LINES!

Reply:
🗑️